torch/utils/cpp_extension.py

```diff
@@ -2091,8 +2088,7 @@ def _jit_compile(name,
     if version > 0:
         if version != old_version and verbose:
             print(f'The input conditions for extension module {name} have changed. ' +
```

Comment:
All these print statements need to be changed to log statement especially fi they aren't going to stderr, right?